app/controllers/phone_numbers_controller.rb

```diff
@@ -1,5 +1,5 @@
 class PhoneNumbersController < ApplicationController
-  load_and_authorize_resource
+  load_and_authorize_resource through: :current_user
```

Comment:
This is great.  Now phone numbers will automatically be associated with the current user.  You don't have to manually set the phone number's user to `current_user` because cancan does that for you this way.